src/main/java/org/opensrp/service/EventService.java

```diff
@@ -170,8 +170,9 @@ public synchronized Event addEvent(Event event, String username) {
 		String planIdentifier = event.getDetails() != null ? event.getDetails().get("planIdentifier") : null;
 		if (isPlanEvaluationEnabled && planIdentifier != null) {
 			PlanDefinition plan = planRepository.get(planIdentifier);
-			if (plan.getStatus().equals(PlanDefinition.PlanStatus.ACTIVE) && (plan.getEffectivePeriod().getEnd() == null
-			        || plan.getEffectivePeriod().getEnd().isAfter(new DateTime().toLocalDate())))
+//			if (plan.getStatus().equals(PlanDefinition.PlanStatus.ACTIVE) && (plan.getEffectivePeriod().getEnd() == null
+//			        || plan.getEffectivePeriod().getEnd().isAfter(new DateTime().toLocalDate())))
+			//TODO : Discuss with Sam
```

Comment:
Pull latest master, this should be reverted

Reply:
Done